lib/client/identity.go

```diff
@@ -86,6 +89,22 @@ func MakeIdentityFile(filePath string, key *Key, format IdentityFileFormat) (err
 		if _, err = output.Write(key.Cert); err != nil {
 			return trace.Wrap(err)
 		}
+		// append trusted host certificate authorities
+		for _, ca := range certAuthorities {
+			for _, publicKey := range ca.GetCheckingKeys() {
+				data, err := sshutils.MarshalAuthorizedHostsFormat(ca.GetClusterName(), publicKey, nil)
+				if err != nil {
+					return trace.Wrap(err)
+				}
+				if _, err = output.Write([]byte(data)); err != nil {
```

Comment:
Should this be prefixed by `\n` in case the previous write didn't add a new line?